### DISCUSSION TITLE
skipping: no hosts matched

### DISCUSSION THREAD

**Discussion by vijayavanisabbiti** [learndevopsonline/learndevopsonline#1639] [Q&A] [answered]
While trying to execute sample ping collection got this error.
help me to understand

<img width="524" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/150422689/a3b7561e-ffd6-4f3b-a492-da791bc6d3c2">

<img width="523" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/150422689/6ceee392-82c1-4c44-b025-16145d418614">

**Comment by sreedharm07** [comment]
Unable to log in into your machine 
Please turn on and ping ip here

**Reply by vijayavanisabbiti** [reply, marked as answer]
I think i got the answer. entered incorrect inventory file name.
